app/views/orga/teams/_form.html.slim

```diff
@@ -80,9 +80,27 @@
     h2 Conference preferences
 
   fieldset.conferences_fieldset
-    = f.simple_fields_for :conference_preferences, f.object.conference_preferences.sort{ |a,b| a.option <=> b.option} do |c|
-      = c.hidden_field :option, value: c.object.option
-      = c.input :conference_id, collection: @conferences.group_by(&:region), as: :grouped_select, group_method: :last, label: c.object.option == 1 ? "Primary choice" : "Secondary Choice", prompt: "Conference", wrapper_html: { class: 'col-md-8' }
+      = f.simple_fields_for :conference_preference do |c|
+        = c.input :first_conference_id, collection: @conferences.group_by(&:region), as: :grouped_select, group_method: :last, label: "Primary choice", prompt: "Conference"
+        = c.input :second_conference_id, collection: @conferences.group_by(&:region), as: :grouped_select, group_method: :last, label: "Secondary choice", prompt: "Conference"
+        = c.input :comment, prompt: "Comments"
+        .form-group
+          label.control-label.col-sm-3 Lightning talk
+          .col-sm-9
+            .checkbox
+              label
+                = c.check_box :lightning_talk
+                |  I would like to give a lightning talk!
+          label.control-label.col-sm-3 Terms
+          .col-sm-9
+            .checkbox
+              label
+                = check_box_tag :terms_ticket, @team.conference_preference.present?, @team.conference_preference.present?
```

Comment:
As said above, when using an acceptance validation for the terms fields, this will simply be an input on the form object itself, same as e.g. the comment.